workers/ohsome_quality_analyst/ohsome/client.py

```diff
@@ -108,50 +111,75 @@ async def get_latest_ohsome_timestamp() -> datetime.datetime:
 
 
 def build_url(
-    layer,
-    endpoint: Optional[str] = None,
+    layer: Layer,
     ratio: bool = False,
-) -> str:
-    """Build endpoint URL of ohsome API."""
-    ohsome_api = OHSOME_API.rstrip("/")
-    if endpoint is None:
-        endpoint = layer.endpoint
-    url = ohsome_api + "/" + endpoint.rstrip("/")
+    group_by: bool = False,
+    contributions: bool = False,
+):
+    if contributions:
+        return OHSOME_API.rstrip("/") + "/contributions/latest/count"
```

Comment:
Currently only "contributions/latest/count" is in use. Should this function be more generell or is this okay. For the latter one there needs to be more docs or a comment.

Reply:
see comment about the variable names

Reply:
I think with the changed variable names this is okay for now.